core/store/src/trie/mem/arena/alloc.rs

```diff
@@ -77,38 +87,32 @@ impl Allocator {
         self.active_allocs_count_gauge.set(self.active_allocs_count as i64);
```

Comment:
Why is this the same information being tracked twice over? Consider getting rid of `active_allocs_bytes` and use the gauges alone for the tracking? See `prometheus::GenericGauge::<_>::add/sub`

Reply:
I did use that before but in parallel tests, two arena instances may be active and they clobber each other's metrics, and leads to flaky tests.

Reply:
Do you mean there is one test that runs multiple threads with arenas at once? If not and you’re speaking of parallelism introduced by `cargo test`, then this is the reason why we only support `cargo nextest` to run the test suite, because we already have other tests like this :)